Makefile

```diff
@@ -1,11 +1,12 @@
 .DEFAULT_GOAL := lint
+DEPLOY_DOWNTIME ?= 0
+DEPLOY_DB_VERSION ?= ""
```

Comment:
I would expect no default here. What is usage of default? (Caller not setting DB dir is a bug in the caller for me.)

Reply:
Fair point, removed in 839db7b

Reply:
Amended to 0249283